src/main/java/com/mcharopkin/server/MainServlet.java

```diff
@@ -30,6 +30,11 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
 		resp.addHeader("Access-Control-Allow-Origin", "192.168.56.101:5080");
 		resp.addHeader("Access-Control-Allow_Methods", "*");
 		resp.setCharacterEncoding("UTF-8");
+		try {
+            		System.out.println("TEST_ERROR");
+       		} catch (Exception ex) {
+            		ex.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.foods.devops.mnscorp.net/coding_rules#rule_key=squid%3AS1148)